core/src/utilities/helpers/generic-helpers.js

```diff
@@ -21,6 +21,10 @@ class GenericHelpersClass {
     return anyParam && this.isFunction(anyParam.then);
   }
 
+  isString(value) {
```

Comment:
Could you add some JSDoc here as well. Also would be great to have a unit test for this one as well. It would be simple but also helps with the test coverage threshold later.